node/src/main/scala/org/bitcoins/node/NeutrinoNode.scala

```diff
@@ -46,7 +46,7 @@ case class NeutrinoNode(
       node <- super.start()
       chainApi <- chainApiFromDb()
       bestHash <- chainApi.getBestBlockHash()
-      _ <- peerMsgSender.sendGetCompactFilterCheckPointMessage(
+      _ <- peerMsgSenders(0).sendGetCompactFilterCheckPointMessage(
```

Comment:
ideally for this and everywhere else you do `peerMsgSenders(0)` we should call upon a random peer